internal/cli/cmd/version/version.go

```diff
@@ -23,9 +23,9 @@ func NewCmd() *cobra.Command {
 	logger := cmdUtil.GetDefaultLoggerInstance()
 	cmd := &cobra.Command{
 		Use:   "version",
-		Short: "Print the version number of oceanbase cli",
+		Short: "Print the version number of OceanBase Cli",
 		Run: func(cmd *cobra.Command, args []string) {
-			logger.Println("Oceanbase cli Version:0.0.1")
+			logger.Println("OceanBase Cli Version: 0.0.1")
```

Comment:
How about inject the version variable? You could refer to variable injection of Dashboard and make the change in another PR.